pkgs/applications/misc/mbrola/default.nix

```diff
@@ -0,0 +1,106 @@
+{ stdenv, fetchurl, unzip, bash, patchelf, stdenv_32bit, makeWrapper, gcc_multi }:
+
+let
+	TCTS="https://tcts.fpms.ac.be/synthesis/mbrola";
```

Comment:
Could you use coding style more consistent with the rest of nixpkgs? Namely:

* Two space indentation (no tabs :cry:)
* A single space on each side of `=`
* No trailing spaces
* Files ending with single `\n` character: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline